### DISCUSSION TITLE
Timelapse deaktivation

### DISCUSSION THREAD

**Discussion by PapsDizzy** [nlef/moonraker-telegram-bot#239] [Q&A]
Hi, im new in Klipper and also new here. 
i have timelapse in mainsail and only use it when i need. 
now i have install the telegram bot and all works fine but the bot is sending me a timelapse after the print. i dont want do have a timelapse just notifications.
how can i deaktivate the telegram-bot timelapse?

**Comment by devbro1** [comment]
I think I found the answer:
https://github.com/nlef/moonraker-telegram-bot/blob/b9b47003e13b8a23af696594aac8ba53c777f5a7/bot/configuration.py#L287

basically remove timelapse section entirely from config and it should deactivate timelapse
